docs/user-guide/cli-using-global-profile-configuration.md

```diff
@@ -165,28 +152,38 @@ For example, to add a new instance of z/OSMF that runs on a different mainframe
         }
     },
     "defaults": {
-        // Change to zosmf_lpar2 if you wish to change default profile
+        // Change to zosmf_lpar2 if you want to change default profile
         "zosmf": "zosmf_lpar1",
         "base": "base"
-    }
+    },
+    "autoStore": true
 }
 ```
 
 You can continue to add more LPARs and more services within each LPAR. After you make changes, save the file and issue a Zowe CLI command to the service to verify the connection.
 
 ## Managing credential security
 
-When you first run the `zowe config init --global-config` command, the `profiles.base.properties.user` and `profiles.base.properties.password` fields are defined to the "secure" array in your configuration file. This ensures that the username and password are stored securely on your computer.
+When you first run the `zowe config init --global-config` command, the `profiles.base.properties.user` and `profiles.base.properties.password` fields are defined to the "secure" array in your configuration file, which help to ensure that the username and password are stored securely on your computer.
 
-To update the secure fields (for example, when you want to change your username and password), issue the `zowe config secure` command.
+To store or update values for the secure fields (for example, when you want to change your username and password), issue the `zowe config secure` command.
```

Comment:
When updating values, I believe you can just press enter to skip a particular property in the case you only want to update a couple properties in a long list. @t1m0thyj can you confirm? If this is the case, @JamesBauman, it might be worth mentioning in the doc.